sources/packages/backend/apps/db-migrations/src/sql/DisbursementOverawards/Create-disbursement-overawards-table.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS sims.disbursement_overawards(
+  id SERIAL PRIMARY KEY,
+  student_id INT NOT NULL REFERENCES sims.students(id),
+  student_assessment_id INT REFERENCES sims.student_assessments(id),
+  disbursement_schedule_id INT REFERENCES sims.disbursement_schedules(id),
+  overaward_value NUMERIC(8, 2) NOT NULL,
+  disbursement_value_code VARCHAR(10) NOT NULL,
+  origin_type sims.disbursement_overaward_origin_types NOT NULL,
+  deleted_at TIMESTAMP WITH TIME ZONE,
```

Comment:
Why does this table require soft delete?

Reply:
Please see https://github.com/bcgov/SIMS/pull/1586#discussion_r1036494827